server/src/views/frame.js

```diff
@@ -222,6 +222,7 @@ class Frame extends React.Component {
 
   clickFullPageButton(e) {
     e.preventDefault();
+    history.pushState({state: "frame"}, "", "#frame");
```

Comment:
The use case I had in mind was specifically sharing the URL with the hash, just like I'd share the URL with the clip ID to link to a clip.  I don't think there's much purpose to changing the hash except to share it.  (I guess it lets you use back/forward, but that's only a minor and questionable feature)